tensorflow_datasets/image/structseg.py

```diff
@@ -0,0 +1,140 @@
+"""
+Organ-at-risk segmentation dataset from head & neck CT scans
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import re
+import tensorflow_datasets.public_api as tfds
+import tensorflow as tf
+import nibabel as nib
```

Comment:
Should be lazy-loaded following instructions here: https://www.tensorflow.org/datasets/add_dataset#extra_dependencies